doc/classes/Quat.xml

```diff
@@ -4,7 +4,7 @@
 		Quaternion.
 	</brief_description>
 	<description>
-		Quaternion is a 4 dimensional vector that is used to represent a rotation. It mainly exists to perform SLERP (spherical-linear interpolation) between two rotations. Multiplying quaternions also cheaply reproduces rotation sequences. However quaternions need to be often renormalized, or else they suffer from precision issues.
+		A 4 dimensional vector that is used to represent a rotation. It mainly exists to perform SLERP (spherical-linear interpolation) between two rotations. Multiplying quaternions also cheaply reproduces rotation sequences. However quaternions need to be often renormalized, or else they suffer from precision issues.
```

Comment:
A 4-dimensional vector representing a rotation.

This needs further rewrite, removing "mainly exists" and "cheaply". Quaternions are a very difficult concept, and we need a nice, clear explanation here.

Reply:
i totally agree